mock-certify-plugin/src/test/java/io/mosip/certify/mock/integration/service/MockVCIssuancePluginTest.java

```diff
@@ -88,4 +89,17 @@ public void getVerifiableCredentialWithLinkedDataProof_withoutSignatureService_t
             Assert.assertEquals("vci_exchange_failed",e.getMessage());
         }
     }
+
+    @Test
+    public void getVerifiableCredentialWithLinkedDataProof_withValidCredentialType() throws VCIExchangeException {
+        JWTSignatureResponseDto jwtSignatureResponseDto = new JWTSignatureResponseDto();
+        jwtSignatureResponseDto.setJwtSignedData("test-jwt");
+        Mockito.when(signatureService.jwtSign(any())).thenReturn(jwtSignatureResponseDto);
+        VCResult vcResult = mockVCIssuancePlugin.getVerifiableCredentialWithLinkedDataProof(vcRequestDto,"holderId",Map.of("accessTokenHash","ACCESS_TOKEN_HASH","client_id","CLIENT_ID"));
```

Comment:
this shouldn't throw an exception, should I wrap this in a try-catch and assert that no exception is thrown?

Reply:
jira task to improve test case added , remove throws from test case.

Reply:
done: https://mosip.atlassian.net/browse/INJICERT-319